server/services/message-service.js

```diff
@@ -20,6 +20,8 @@ async function openConnections () {
 
 async function publishClaim (claim) {
   try {
+    console.log('calculationSender connected', calculationSender.isConnected())
+    console.log('scheduleSender connected', scheduleSender.isConnected())

```

Comment:
I've left the logging in to provide additional information on the status of the system when the claim is published